apps/wallet/src/ui/app/background-client/index.ts

```diff
@@ -271,6 +271,17 @@ export class BackgroundClient {
         );
     }
 
+    public async deriveNextAccount() {
+        return await lastValueFrom(
```

Comment:
why the await

Reply:
yeah it's not necessary, just copied-pasted from the function above. We should refactor this when we change the way we communicate with the background service